reference/settings.v3.yml

```
@@ -2,6 +2,30 @@ openapi: '3.0.3'
 info:
   description: |-
     Manage settings and configuration for BigCommerce hosted stores and headless storefronts.
+
+    ## Available settings and configuration
+    | Resource                   | Description                                                                   |
+    |:---------------------------|:------------------------------------------------------------------------------|
+    | Analytics                  | Identifies and updates Web analytics providers.                               |
+    | Catalog                    | Identifies and updates catalog settings.                                      |
+    | Email Statuses             | Identifies and updates transactional email settings.                          |
+    | Favicon Image              | Creates a favicon image.                                                      |
+    | Inventory                  | Identifies and updates inventory settings.                                    |
+    | Inventory Notifications    | Identifies and updates inventory notifications.                               |
+    | Logo                       | Identifies and updates logo settings.                                         |
+    | Logo Image                 | Creates a logo image.                                                         |
+    | Search Filters             | Identifies and updates enabled filters, available filters, and contextual filters.|
+    | Store Locale               | Identifies and updates locale settings.                                        |
+    | Store Profile              | Identifies and updates store profile.                                          |
+    | Storefront Category        | Identifies and updates storefront category settings.                           |
+    | Storefront Product         | Identifies and updates storefront product settings.                            |
+    | Storefront Robotstxt       | Identifies and updates storefront Robots.txt settings.                         |
```

```suggestion
    | Storefront Robots.txt       | Identifies and updates storefront Robots.txt settings.                         |
```
Does it make sense to update the page section title as well?